Promact.Oauth.Server/src/Promact.Oauth.Server/Controllers/UserController.cs

```diff
@@ -204,192 +238,265 @@ public IActionResult GetRole()
                     foreach (var error in apiEx.Errors)
                         _logger.LogError("Forgot Password mail not send " + error);
                 }
-                throw apiEx;
+                return BadRequest();
             }
             catch (ArgumentNullException argEx)
```

Comment:
If SendGridApi Key and From values are not set in appsettings then we throw an ArgumentNullException.